examples/resources/spectrocloud_workspace/resource.tf

```diff
@@ -0,0 +1,80 @@
+data "spectrocloud_cluster" cluster1 {
+  name = "vsphere-picard-2"
+}
+
+resource "spectrocloud_workspace" "workspace" {
+  name = "wsp-tf"
+
+  clusters {
+    name = data.spectrocloud_cluster.cluster1.name
+    uid = data.spectrocloud_cluster.cluster1.id
```

Comment:
@nikchern, Should we just take uid as input?

Reply:
@rishi-anand filed a bug to support it, as of now both needed.